build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -55,6 +55,19 @@ spec:
                   minimum: 1
                   maximum: 65535
                   default: 179
+                confederation:
+                  type: object
+                  properties:
+                    enable:
+                      type: boolean
+                      default: false
+                    identifier:
+                      type: integer
+                      format: int32
```

Comment:
what is the range of valid values?

Reply:
As per the discussion in the other PR, we will keep using range 1-65535 for ASNs in this version of the API, and extend to the full range in the next version, as it is not a backward compatible change.